syn/python/build_translated_names.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import argparse
+
+from flow_utils import *
+
+def main():
+    arg_parser = argparse.ArgumentParser(
+        description=
+"""Create yosys script to translate cell names from given timing reports to human readable names
+(assumes flatted synthesis run)""")
```

Comment:
Can you shorten this line, and run the file through yapf with our opentitan configuration?

Reply:
yapf needs a bit of help here, manually split the line into two and re-running yapf should do the trick.